### PR TITLE
README

### PR DESCRIPTION
**Pull request by JenHynes**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,42 @@
 # CSCI 420 (Spring 2021) MontyPython UML Editor
 
+This command line program creates a text representation of a UML diagram though user input. The program’s capabilities include adding, deleting, displaying, and modifying UML diagram elements. It also supports the ability to save and load diagrams.
+
+## Prerequisites
+1. To run the program, [Python 3.9.1](https://www.python.org/downloads/) needs to be installed.
+1. During installation, check the box labeled "Add Python 3.9 to PATH" so Python programs can be run by typing `Python` in a terminal.
```
Comment (sfmalloy) [inline README.md]: Lowercase `python` here at the end of the sentence since the default command name is lowercase.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,42 @@
...
+
+## Running the program
+1. Download the program from our [repository](https://github.com/mucsci-students/2021sp-420-MontyPython).
+1. In a terminal, navigate to the directory the repository download has been saved in.
+1. Type `Python REPL.py` to launch the command line program.
```
Comment (sfmalloy) [inline README.md]: Same thing with lowercase `python` here.

File: README.md [hunk 3]
```diff
@@ -1,2 +1,42 @@
 # CSCI 420 (Spring 2021) MontyPython UML Editor
 
+This command line program creates a text representation of a UML diagram though user input. The program’s capabilities include adding, deleting, displaying, and modifying UML diagram elements. It also supports the ability to save and load diagrams.
+
+## Prerequisites
+1. To run the program, [Python 3.9.1](https://www.python.org/downloads/) needs to be installed.
```
Comment (sfmalloy) [inline README.md:6]: Change the numbers so not everything is step 1.

**Review by sfmalloy — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (JenHynes) [reply README.md:6]: That's GitHub markdown syntax for ordered lists

Comment (sfmalloy) [reply README.md:6]: > That's GitHub markdown syntax for ordered lists

Ah I see now, my bad.

**Review by sfmalloy — APPROVED** [review]
Looks good!

**Pull request merged**